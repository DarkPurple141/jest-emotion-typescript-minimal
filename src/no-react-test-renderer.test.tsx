import React from 'react';
import Component from './app';
import snapshotDiff from 'snapshot-diff';

/* this doesn't work (it doesn't serialize) */
test('snapshot difference using snapshotdiff directly', () => {
  expect(snapshotDiff(<Component />, <Component isError/>)).toMatchSnapshot();
});
