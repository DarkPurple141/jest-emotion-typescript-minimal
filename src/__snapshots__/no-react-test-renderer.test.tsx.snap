// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot difference using snapshotdiff directly 1`] = `
"Snapshot Diff:
- <App />
+ <App isError={true} />

@@ -3,10 +3,10 @@
      className=\\"css-gj8yp6-paragraphStyles\\"
    >
      Hello
    </p>
    <button
-     className=\\"css-qrvbnj-buttonStyles\\"
+     className=\\"css-1d3nsau-errorStyles\\"
    >
      World
    </button>
  </div>"
`;
